thesis-api/main/main.py

```diff
@@ -1,18 +1,23 @@
 import logging
 
-from fastapi import FastAPI
+from fastapi import FastAPI, Depends
+from starlette.status import HTTP_200_OK
+
+from services.category_service import CategoryService
+from web.schemas import CategoryListResponse, CategoryResponse
 
 app = FastAPI()
 
 
 logging.getLogger().setLevel(logging.INFO)
 
 
-@app.get("/")
-async def root():
-    return {"message": "Hello World"}
+@app.get("/api/categories", status_code=HTTP_200_OK, response_model=CategoryListResponse)
+async def get_categories(service: CategoryService = Depends()):
+    categories = service.get_all()
 
+    res = []
```

Comment:
Can be shortened to:
```python
res = [CategoryResponse(id=c.id, name=c.name) for c in categories]
```

Reply:
And add at least an empty file in `test` directory, because if `test` directory does not exist, GitHub checks fail.